benchexec/tools/ceagle.py

```diff
@@ -24,19 +25,8 @@
 
 class Tool(benchexec.tools.template.BaseTool):
 
-    REQUIRED_PATHS = [
-                  "ceagle.sh",
-                  "dfs.py",
-                  "parse2str.py",
-                  "parsece.sh",
-                  "svcore",
-                  "svie",
-                  "verifier.py",
-                  "z3"
-                  ]
-
     def executable(self):
-        return util.find_executable('ceagle.sh')
+        return util.find_executable('sv-ceagle')
```

Comment:
This change is necessary, since we use a wrapper shell script for making the output following svcomp convention and preprocessing files last year, but this year we implement all these inside our binary.

I understand your concern, so how about we re-submit a new module called ceagle_sv this year without changing previous ones?